### ISSUE TITLE
v0.6.0: Exception thrown when opening config in backend

### ISSUE THREAD

**Issue by dzschille** [justbetter/magento2-sentry#14]
I just installed the module for the first time. The installation had no errors but when i open the module config in the backend (path admin/system_config/edit/section/sentry/) i get this error message:

```
1 exception(s):
Exception #0 (ReflectionException): Class JustBetter\Sentry\Model\Config\Source\ScriptTagPlacement does not exist

Exception #0 (ReflectionException): Class JustBetter\Sentry\Model\Config\Source\ScriptTagPlacement does not exist
#0 /var/www/public/vendor/magento/framework/Code/Reader/ClassReader.php(19): ReflectionClass->__construct('JustBetter\\Sent...')
#1 /var/www/public/vendor/magento/framework/ObjectManager/Definition/Runtime.php(44): Magento\Framework\Code\Reader\ClassReader->getConstructor('JustBetter\\Sent...')
#2 /var/www/public/vendor/magento/framework/ObjectManager/Factory/Compiled.php(100): Magento\Framework\ObjectManager\Definition\Runtime->getParameters('JustBetter\\Sent...')
#3 /var/www/public/vendor/magento/framework/ObjectManager/ObjectManager.php(70): Magento\Framework\ObjectManager\Factory\Compiled->create('JustBetter\\Sent...')
#4 /var/www/public/vendor/magento/module-config/Model/Config/SourceFactory.php(37): Magento\Framework\ObjectManager\ObjectManager->get('JustBetter\\Sent...')
#5 /var/www/public/vendor/magento/module-config/Model/Config/Structure/Element/Field.php(440): Magento\Config\Model\Config\SourceFactory->create('JustBetter\\Sent...')
#6 /var/www/public/vendor/magento/module-config/Model/Config/Structure/Element/Field.php(376): Magento\Config\Model\Config\Structure\Element\Field->_getOptionsFromSourceModel('JustBetter\\Sent...')
#7 /var/www/public/vendor/magento/module-config/Block/System/Config/Form.php(404): Magento\Config\Model\Config\Structure\Element\Field->getOptions()
#8 /var/www/public/vendor/magento/module-config/Block/System/Config/Form.php(325): Magento\Config\Block\System\Config\Form->_initElement(Object(Magento\Config\Model\Config\Structure\Element\Field\Interceptor), Object(Magento\Framework\Data\Form\Element\Fieldset), 'sentry/general/...', '', '')
#9 /var/www/public/vendor/magento/module-config/Block/System/Config/Form.php(265): Magento\Config\Block\System\Config\Form->initFields(Object(Magento\Framework\Data\Form\Element\Fieldset), Object(Magento\Config\Model\Config\Structure\Element\Group\Interceptor), Object(Magento\Config\Model\Config\Structure\Element\Section))
#10 /var/www/public/vendor/magento/module-config/Block/System/Config/Form.php(211): Magento\Config\Block\System\Config\Form->_initGroup(Object(Magento\Config\Model\Config\Structure\Element\Group\Interceptor), Object(Magento\Config\Model\Config\Structure\Element\Section), Object(Magento\Framework\Data\Form))
#11 /var/www/public/vendor/magento/module-config/Block/System/Config/Form.php(548): Magento\Config\Block\System\Config\Form->initForm()
#12 /var/www/public/vendor/magento/framework/View/Element/AbstractBlock.php(666): Magento\Config\Block\System\Config\Form->_beforeToHtml()
#13 /var/www/public/vendor/magento/framework/View/Layout.php(557): Magento\Framework\View\Element\AbstractBlock->toHtml()
#14 /var/www/public/vendor/magento/framework/View/Layout.php(533): Magento\Framework\View\Layout->_renderBlock('system\\config\\f...')
#15 /var/www/public/vendor/magento/framework/View/Layout.php(488): Magento\Framework\View\Layout->renderNonCachedElement('system\\config\\f...')
#16 /var/www/public/vendor/magento/framework/View/Element/AbstractBlock.php(507): Magento\Framework\View\Layout->renderElement('system\\config\\f...', true)
#17 /var/www/public/vendor/magento/module-config/view/adminhtml/templates/system/config/edit.phtml(22): Magento\Framework\View\Element\AbstractBlock->getChildHtml('form')
#18 /var/www/public/vendor/magento/framework/View/TemplateEngine/Php.php(59): include('/var/www/public...')
#19 /var/www/public/vendor/magento/framework/View/Element/Template.php(270): Magento\Framework\View\TemplateEngine\Php->render(Object(Magento\Config\Block\System\Config\Edit), '/var/www/public...', Array)
#20 /var/www/public/vendor/magento/framework/View/Element/Template.php(300): Magento\Framework\View\Element\Template->fetchView('/var/www/public...')
#21 /var/www/public/vendor/magento/module-backend/Block/Template.php(126): Magento\Framework\View\Element\Template->_toHtml()
#22 /var/www/public/vendor/magento/framework/View/Element/AbstractBlock.php(667): Magento\Backend\Block\Template->_toHtml()
#23 /var/www/public/vendor/magento/framework/View/Layout.php(557): Magento\Framework\View\Element\AbstractBlock->toHtml()
#24 /var/www/public/vendor/magento/framework/View/Layout.php(533): Magento\Framework\View\Layout->_renderBlock('system.config.e...')
#25 /var/www/public/vendor/magento/framework/View/Layout.php(488): Magento\Framework\View\Layout->renderNonCachedElement('system.config.e...')
#26 /var/www/public/vendor/magento/framework/View/Layout.php(585): Magento\Framework\View\Layout->renderElement('system.config.e...', false)
#27 /var/www/public/vendor/magento/framework/View/Layout.php(535): Magento\Framework\View\Layout->_renderContainer('content', false)
#28 /var/www/public/vendor/magento/framework/View/Layout.php(488): Magento\Framework\View\Layout->renderNonCachedElement('content')
#29 /var/www/public/vendor/magento/framework/View/Layout.php(585): Magento\Framework\View\Layout->renderElement('content', false)
#30 /var/www/public/vendor/magento/framework/View/Layout.php(535): Magento\Framework\View\Layout->_renderContainer('admin.scope.col...', false)
#31 /var/www/public/vendor/magento/framework/View/Layout.php(488): Magento\Framework\View\Layout->renderNonCachedElement('admin.scope.col...')
#32 /var/www/public/vendor/magento/framework/View/Layout.php(585): Magento\Framework\View\Layout->renderElement('admin.scope.col...', false)
#33 /var/www/public/vendor/magento/framework/View/Layout.php(535): Magento\Framework\View\Layout->_renderContainer('main.col', false)
#34 /var/www/public/vendor/magento/framework/View/Layout.php(488): Magento\Framework\View\Layout->renderNonCachedElement('main.col')
#35 /var/www/public/vendor/magento/framework/View/Layout.php(585): Magento\Framework\View\Layout->renderElement('main.col', false)
#36 /var/www/public/vendor/magento/framework/View/Layout.php(535): Magento\Framework\View\Layout->_renderContainer('page.main.conta...', false)
#37 /var/www/public/vendor/magento/framework/View/Layout.php(488): Magento\Framework\View\Layout->renderNonCachedElement('page.main.conta...')
#38 /var/www/public/vendor/magento/framework/View/Layout.php(585): Magento\Framework\View\Layout->renderElement('page.main.conta...', false)
#39 /var/www/public/vendor/magento/framework/View/Layout.php(535): Magento\Framework\View\Layout->_renderContainer('page.content', false)
#40 /var/www/public/vendor/magento/framework/View/Layout.php(488): Magento\Framework\View\Layout->renderNonCachedElement('page.content')
#41 /var/www/public/vendor/magento/framework/View/Layout.php(585): Magento\Framework\View\Layout->renderElement('page.content', false)
#42 /var/www/public/vendor/magento/framework/View/Layout.php(535): Magento\Framework\View\Layout->_renderContainer('page.wrapper', false)
#43 /var/www/public/vendor/magento/framework/View/Layout.php(488): Magento\Framework\View\Layout->renderNonCachedElement('page.wrapper')
#44 /var/www/public/vendor/magento/framework/View/Layout.php(585): Magento\Framework\View\Layout->renderElement('page.wrapper', false)
#45 /var/www/public/vendor/magento/framework/View/Layout.php(535): Magento\Framework\View\Layout->_renderContainer('backend.page', false)
#46 /var/www/public/vendor/magento/framework/View/Layout.php(488): Magento\Framework\View\Layout->renderNonCachedElement('backend.page')
#47 /var/www/public/vendor/magento/framework/View/Layout.php(585): Magento\Framework\View\Layout->renderElement('backend.page', false)
#48 /var/www/public/vendor/magento/framework/View/Layout.php(535): Magento\Framework\View\Layout->_renderContainer('root', false)
#49 /var/www/public/vendor/magento/framework/View/Layout.php(488): Magento\Framework\View\Layout->renderNonCachedElement('root')
#50 /var/www/public/vendor/magento/framework/View/Layout.php(954): Magento\Framework\View\Layout->renderElement('root')
#51 /var/www/public/generated/code/Magento/Framework/View/Layout/Interceptor.php(37): Magento\Framework\View\Layout->getOutput()
#52 /var/www/public/vendor/magento/framework/View/Result/Page.php(257): Magento\Framework\View\Layout\Interceptor->getOutput()
#53 /var/www/public/vendor/magento/framework/View/Result/Layout.php(170): Magento\Framework\View\Result\Page->render(Object(Magento\Framework\App\Response\Http\Interceptor))
#54 /var/www/public/generated/code/Magento/Backend/Model/View/Result/Page/Interceptor.php(24): Magento\Framework\View\Result\Layout->renderResult(Object(Magento\Framework\App\Response\Http\Interceptor))
#55 /var/www/public/vendor/magento/framework/App/Http.php(139): Magento\Backend\Model\View\Result\Page\Interceptor->renderResult(Object(Magento\Framework\App\Response\Http\Interceptor))
#56 /var/www/public/vendor/magento/framework/App/Bootstrap.php(257): Magento\Framework\App\Http->launch()
#57 /var/www/public/pub/index.php(37): Magento\Framework\App\Bootstrap->run(Object(Magento\Framework\App\Http\Interceptor))
#58 {main}

```
The error seems to be the [new field script_tag_placement](https://github.com/justbetter/magento2-sentry/commit/d24cc63de59e5f7a2f9090400c46fe1741db6c86#diff-c4a53edec4557757b0db215231c89c32R34). When i remove it from justbetter/magento2-sentry/etc/adminhtml/system.xml and clear the cache, the error is gone.

**Comment by royduin** [comment]
@rakibabu and @michielgerritsen

**Comment by rakibabu** [comment]
HI @dzschille ,

Thanks for posting this issue. It has been fixed in release v0.6.1 (https://github.com/justbetter/magento2-sentry/tree/0.6.1) with PR (https://github.com/justbetter/magento2-sentry/pull/16).

**Comment by dzschille** [comment]
That was really quick! Thanks!!
I updated to v0.6.1 and it works well now.
